sw-toolbox.js

```diff
@@ -23,7 +23,104 @@
 },{}],4:[function(require,module,exports){
 "use strict";var url=new URL("./",self.location),basePath=url.pathname,pathRegexp=require("path-to-regexp"),Route=function(e,t,i,s){0!==t.indexOf("/")&&(t=basePath+t),this.method=e,this.keys=[],this.regexp=pathRegexp(t,this.keys),this.options=s,this.handler=i};Route.prototype.makeHandler=function(e){var t=this.regexp.exec(e),i={};return this.keys.forEach(function(e,s){i[e.name]=t[s+1]}),function(e){return this.handler(e,i,this.options)}.bind(this)},module.exports=Route;
 },{"path-to-regexp":12}],5:[function(require,module,exports){
-"use strict";function regexEscape(t){return t.replace(/[-\/\\^$*+?.()|[\]{}]/g,"\\$&")}var Route=require("./route"),keyMatch=function(t,e){for(var r=Object.keys(t),o=0;o<r.length;o++){var u=new RegExp(r[o]);if(u.test(e))return t[r[o]]}return null},Router=function(){this.routes={},this["default"]=null};["get","post","put","delete","head","any"].forEach(function(t){Router.prototype[t]=function(e,r,o){return this.add(t,e,r,o)}}),Router.prototype.add=function(t,e,r,o){o=o||{};var u=o.origin||self.location.origin;u=u instanceof RegExp?u.source:regexEscape(u),t=t.toLowerCase();var n=new Route(t,e,r,o);this.routes[u]=this.routes[u]||{},this.routes[u][t]=this.routes[u][t]||{},this.routes[u][t][n.regexp.source]=n},Router.prototype.matchMethod=function(t,e){e=new URL(e);var r=e.origin,o=e.pathname;t=t.toLowerCase();var u=keyMatch(this.routes,r);if(!u)return null;var n=u[t];if(!n)return null;var a=keyMatch(n,o);return a?a.makeHandler(o):null},Router.prototype.match=function(t){return this.matchMethod(t.method,t.url)||this.matchMethod("any",t.url)},module.exports=new Router;
+/*
```

Comment:
Uglify doesn't like `for ... of`, but it fails by just not uglifying, which I can live with.